pupil_src/shared_modules/gaze_producer/model/calibration_storage.py

```diff
@@ -179,6 +170,7 @@ def _load_calibration_from_file(self, file_name):
 
     def _load_recorded_calibrations(self):
         notifications = fm.load_pldata_file(self._rec_dir, "notify")
+        counter = 1
         for topic, data in zip(notifications.topics, notifications.data):
```

Comment:
```suggestion
        topics_and_data = zip(notifications.topics, notifications.data)
        for counter, (topic, data) in enumerate(topics_and_data, start=1):
```

Reply:
There's two `continue`s in the loop. `enumerate()` would give us notification number, not the calibration number, if I see the correctly?

Reply:
Correct. Github had part of the loop collapsed.